runtime/core/frontend/fbank.h

```diff
@@ -24,13 +24,43 @@
 #include "frontend/fft.h"
 #include "utils/log.h"
 
+#define S16_TO_FLOAT_SCALE 32768
+
 namespace wenet {
 
 // This code is based on kaldi Fbank implementation, please see
 // https://github.com/kaldi-asr/kaldi/blob/master/src/feat/feature-fbank.cc
+
+enum class WindowType {
```

Comment:
`Enum` should be named as `kPovey`, `kHanning` in our(google c++) style convention, please see
https://google.github.io/styleguide/cppguide.html#Enumerator_Names